google-cloud-bigtable-bom/pom.xml

```diff
@@ -94,6 +94,11 @@
           <artifactId>proto-google-cloud-bigtable-v2</artifactId>
           <version>2.8.1-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-bigtable-v2:current} -->
         </dependency>
+          <dependency>
+              <groupId>com.google.cloud</groupId>
+              <artifactId>google-cloud-bigtable-stats</artifactId>
+              <version>2.8.1-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-bigtable-v2:current} -->
```

Comment:
I think you want {x-version-update:google-cloud-bigtable:current} or to create your own